### PR TITLE
✨(datasources) add datasource configuration for edxapp mysql database

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

To get general platform statistics such as number of users, courses, enrollments... the database from edxapp has to be configured as a datasource.

# Proposal

- [x] add configuration for mysql edxapp (from learning-analytics-playground)
- [x] default dashboard to test consistency (number of users)

platform_user

### CODE REVIEW

**Review by jmaupetit — COMMENTED** [review]
Yay :tada: 

Can you post a screenshot of this new Dashboard?

File: etc/grafana/provisioning/datasources/mysql.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: 1
```
[the remark this answers is not in the document]
Comment (quitterie-lcs) [reply etc/grafana/provisioning/datasources/mysql.yaml:1]: I would prefer keep `mysql.yaml`. IMO, such files define the type of datasources, if we need several of mysql datasources, we can list them in this config file

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**